src/components/EventCard/EventCard.js

```diff
@@ -43,7 +44,7 @@ const EventCard = ({
   };
 
   return (
-    <div className="EventCard">
+    <div className={`EventCard ${className ? className : ``}`}>
```

Comment:
This could be simpler and written as `${className}` instead of `${className ? className : ''}` if the prop defaulted to an empty string (i.e. `className = ''` in line 11).